RevenueCatUI/CustomerCenter/Views/RestorePurchasesAlert.swift

```diff
@@ -31,6 +31,8 @@ struct RestorePurchasesAlert: ViewModifier {
     @Environment(\.openURL)
     var openURL
 
+    @EnvironmentObject private var customerCenterViewModel: CustomerCenterViewModel
```

Comment:
Made this an environment object for ease of access.

Reply:
ohhh nice